frontend/src/metabase-lib/Question.ts

```diff
@@ -1129,6 +1132,14 @@ class QuestionInner {
     }
   }
 
+  setParameter(newParameter) {
```

Comment:
nit: should this function parameter be called just `parameter` because the next function `setParameters` has its parameter (so much parameters 😵‍💫) named `parameters`, or at least change the next function's one to `newParameters` to match them.